api/bazel/repository_locations.bzl

```diff
@@ -1,11 +1,11 @@
 BAZEL_SKYLIB_RELEASE = "0.9.0"
 BAZEL_SKYLIB_SHA256 = "1dde365491125a3db70731e25658dfdd3bc5dbdfd11b840b3e987ecf043c7ca0"
 
-OPENCENSUS_PROTO_GIT_SHA = "5cec5ea58c3efa81fa808f2bd38ce182da9ee731"  # Jul 25, 2019
-OPENCENSUS_PROTO_SHA256 = "faeb93f293ff715b0cb530d273901c0e2e99277b9ed1c0a0326bca9ec5774ad2"
+OPENCENSUS_PROTO_GIT_SHA = "be218fb6bd674af7519b1850cdf8410d8cbd48e8"  # Dec 20, 2019
+OPENCENSUS_PROTO_SHA256 = "e3bbdc94375e86c0edfb2fc5851507e08a3f26ee725ffff7c5c0e73264bdfcde"
 
-PGV_GIT_SHA = "de8fa833aeb04a6bf84c313e39898c22f381fb05"  # Nov 11,2019
-PGV_SHA256 = "c53d1a4af3c48313de174244a98b78aa05dd1f81f92bbf426c2417aea0060d1a"
+PGV_GIT_SHA = "6d6fa03f9dc67580afdf6f43d6b20e8ebc4fdf6c"  # Dec 18,2019
```

Comment:
Envoy just bumped bazel to 2.0.0 where `maven_jar` is deprecated.
Might need to bump PGV to include the fix for that in a later commit https://github.com/envoyproxy/protoc-gen-validate/pull/304

Reply:
```diff
-PGV_GIT_SHA = "de8fa833aeb04a6bf84c313e39898c22f381fb05"  # Nov 11,2019
-PGV_SHA256 = "c53d1a4af3c48313de174244a98b78aa05dd1f81f92bbf426c2417aea0060d1a"
+PGV_GIT_SHA = "973ea075fe989fd7878619845d1e532e5bfe7115"  # Dec 26, 2019
+PGV_SHA256 = "d1cd20dd2cae953d6ccd09db6926fe67d0d2231412666c9b4ac5f159011f14a6"
```

Reply:
Thanks, done